<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizCombineNotesDialog</class>
 <widget class="QDialog" name="WizCombineNotesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>403</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Combine Notes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QListWidget" name="listNotes">
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnMoveUp">
         <property name="text">
          <string>Move Up</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMoveDown">
         <property name="text">
          <string>Move Down</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnPreview">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>New note title:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editNewTitle"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Splitter HTML:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editSplitter"/>
   </item>
   <item>
    <widget class="QCheckBox" name="btnAddTitle">
     <property name="text">
      <string>Add note title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listNotes</tabstop>
  <tabstop>btnMoveUp</tabstop>
  <tabstop>btnMoveDown</tabstop>
  <tabstop>btnPreview</tabstop>
  <tabstop>editNewTitle</tabstop>
  <tabstop>editSplitter</tabstop>
  <tabstop>btnAddTitle</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WizCombineNotesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WizCombineNotesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
